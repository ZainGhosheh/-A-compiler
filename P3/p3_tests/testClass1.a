m: custom {
    x: int;
    y: int;
    z: int = 1;
    f : () -> void {
        a: int;
        b: int;
        while (true) {
            while (false) {
                while (true) {
                    return;
                }
            }
        }
        if (true) {
            if (false) {
            }
        }
    }
};
