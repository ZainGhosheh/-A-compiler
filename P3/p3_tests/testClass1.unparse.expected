m: custom {
    x: int;
    f : () -> void {
        a: int;
        b: int;
    }
};
